general/administration/migrate-from-emby.md

```diff
@@ -11,7 +11,8 @@ The original procedure is provided below for reference however we cannot support
 
 ## Unofficial procedure
 
-~~Jellyfin offers a seamless migration from Emby version 3.5.2 or earlier. Emby versions 3.5.3 or 3.6+ cannot be easily migrated, and we recommend rebuilding your library instead.~~
+> [!NOTE]
+> Jellyfin offers a seamless migration from Emby version 3.5.2 or earlier. Emby versions 3.5.3 or 3.6+ cannot be easily migrated, and we recommend rebuilding your library instead.
```

Comment:
don't change it like this :) we've explicitly crossed out this sentence because migrating instead of creating a new config causes subtle issues down the road.

Reply:
Ah, I mistook that for bold instead of strikethrough, good catch.

Reply:
Might be better to write that instead. Due to subtle bugs introduced, migration is no longer advised or supported

Reply:
```suggestion
> While it is technically possible to migrate existing configuration of Emby version 3.5.2 or earlier, due to subtle and weird bugs reported after such attempts we do not recommend this migration. Emby versions 3.5.3 or 3.6+ cannot be migrated. Thus we recommend creating a new Jellyfin configuration and rebuilding your library instead.
```